### DISCUSSION TITLE
Public general EU channels removed from all lists

### DISCUSSION THREAD

**Discussion by vpetruzziello** [iptv-org/community#319] [Q&A] [answered]
Dear all, maybe I was not so good to find the answers, from some weeks I noticed some channels have been removed from lists ( Koper Capodistria, France 2/3, NPO 1/2/3 and many other public general european channel).  Will these channels by any chance be available again soon ? Thank you in advance for your help and attention

**Comment by BellezaEmporium** [comment, marked as answer]
It might come back due to a bug we've found in the treatment of the streams.
